R/steps-rcmdcheck.R

```diff
@@ -102,9 +104,30 @@ RCMDcheck <- R6Class(
 #' @export
 step_rcmdcheck <- function(...,
                            warnings_are_errors = NULL, notes_are_errors = NULL,
-                           args = c("--no-manual", "--as-cran"),
-                           build_args = "--force", error_on = "warning",
+                           args = NULL, build_args = NULL, error_on = "warning",
                            repos = repo_default(), timeout = Inf) {
+  if (quo_is_null(build_args)) {
```

Comment:
You need to `enquo(build_args)` here. I'm seriously wondering why current `master` even works -- I think the `RCMDcheck$new()` is missing many `eval_tidy(enquo())` calls.

Reply:
Oh -- the evaluation is done in `add_step()`. This makes more sense.

In this case, a simple `is.null()` should do the job. (And maybe we need a comment in the code to avoid future WATs.)

Reply:
Haven't really used `quo()` and `enquo()` in detail before. Feel free to finish up here and saving me hours of fiddling :sweat_smile: